src/components/elements/typography.ts

```diff
@@ -85,7 +100,13 @@ export const Text = styled.p`
 /////////////////// TEXT IS DEPRECATED. PLEASE USE HEADER. ///////////////////
 //////////////////////////////////////////////////////////////////////////////
 
-export const Header = styled(({ as = 'h2', children, ...props }) =>
+type HeaderProps = {
+    as?: string
+    children?: React.ReactNode
+    props?: TextProps
```

Comment:
do we have to add `props` here? 
since you are adding everything from `TextProps` to the type, I think we omit this. 
If I'm missing something, please let me know, thanks thanks :pray:

Reply:
We might need to work on our storybook later. It's answer for our team
Thanks for good question, I thought the same 🙇

Reply:
Right. Also decided to merge `TextProps` to `BaseElementProps`
Thanks 🙇